<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTextBoxes</class>
 <widget class="QWidget" name="FormTextBoxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTextBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextBoxes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxContent">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Content</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelText">
        <property name="text">
         <string>&amp;Text</string>
        </property>
        <property name="buddy">
         <cstring>plainTextEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFont">
        <property name="text">
         <string>&amp;Font</string>
        </property>
        <property name="buddy">
         <cstring>fontComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPointSize">
        <property name="text">
         <string>&amp;Point size</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxPointSize</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxPointSize">
        <property name="suffix">
         <string>pt</string>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelStyle">
        <property name="text">
         <string>&amp;Style</string>
        </property>
        <property name="buddy">
         <cstring>checkBoxBold</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxBold">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxItalic">
          <property name="text">
           <string>Italic</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelAlignment">
        <property name="text">
         <string>Alignment</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelHorizontalAlignment">
            <property name="text">
             <string>&amp;Horizontal</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxHorizontalAlignment</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelVerticalAlignment">
            <property name="text">
             <string>&amp;Vertical</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxVerticalAlignment</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxHorizontalAlignment"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxVerticalAlignment"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Margin</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxMargin</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxMargin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBackgroundColor">
        <property name="text">
         <string>Back&amp;ground color</string>
        </property>
        <property name="buddy">
         <cstring>pushButtonBackgroundColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelTextColor">
        <property name="text">
         <string>Text &amp;color</string>
        </property>
        <property name="buddy">
         <cstring>pushButtonTextColor</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelBorders">
        <property name="text">
         <string>&amp;Borders</string>
        </property>
        <property name="buddy">
         <cstring>checkBoxHasBorders</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBoxHasBorders">
        <property name="text">
         <string>Visible</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelBorderRadius">
        <property name="text">
         <string>Border &amp;radius</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxBorderRadius</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinBoxBorderRadius">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelBorderWidth">
        <property name="text">
         <string>Border &amp;width</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxBorderWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxBorderWidth">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ColorButton" name="pushButtonBackgroundColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="pushButtonTextColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBorderColor">
        <property name="text">
         <string>B&amp;order color</string>
        </property>
        <property name="buddy">
         <cstring>pushButtonBorderColor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ColorButton" name="pushButtonBorderColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonSaveTheme">
       <property name="toolTip">
        <string>Save as default theme</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/icons/save-theme-icon.png</normaloff>:/icons/icons/save-theme-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApplyTheme">
       <property name="toolTip">
        <string>Apply default theme</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/icons/apply-theme-icon.png</normaloff>:/icons/icons/apply-theme-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>Classes/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
